jax/experimental/jax2tf/jax_export.py

```diff
@@ -588,19 +620,23 @@ def _check_lowering(lowering) -> None:
 
 def _check_module(mod: ir.Module, *,
                   allow_non_replicated_sharding: bool,
-                  allow_all_custom_calls: bool):
+                  disabled_checks: Sequence[DisabledSafetyCheck]) -> None:
   """Run a number of checks on the module.
 
   Args:
     allow_non_replicated_sharding: whether the module is allowed to contain
       non_replicated sharding annotations.
-    allow_all_custom_calls: whether we should allow all custom calls, or
-      only those who we have explicitly marked as stable.
+    disabled_checks: the safety checks that are disabled.
   """
   sharding_attr = ir.StringAttr.get("Sharding", mod.context)
+  allowed_custom_call_targets: List[str] = list(_CUSTOM_CALL_TARGETS_GUARANTEED_STABLE)
+  for dc in disabled_checks:
+    target = dc.is_custom_call()
+    if target is not None:
+      allowed_custom_call_targets.append(target)
   allowed_custom_call_targets_attrs = [
```

Comment:
I don't know if `ir.StringAttr` is hashable in Python, but if so, we can use set here so that we avoid linear search in line 663?

Reply:
It is hashable, and I changed this to use sets.